specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2021-03-01-preview/cosmos-db.json

```diff
@@ -0,0 +1,8959 @@
+          }
+        },
+        "description": "Gets the RUs per second of the MongoDB database under an existing Azure Cosmos DB database account with the provided name.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/databaseNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The RUs per second of the MongoDB database was retrieved successfully.",
+            "schema": {
+              "$ref": "#/definitions/ThroughputSettingsGetResults"
+            }
+          }
+        }
+      },
+      "put": {
+        "operationId": "MongoDBResources_UpdateMongoDBDatabaseThroughput",
+        "x-ms-examples": {
+          "CosmosDBMongoDBDatabaseThroughputUpdate": {
+            "$ref": "./examples/CosmosDBMongoDBDatabaseThroughputUpdate.json"
+          }
+        },
+        "description": "Update RUs per second of the an Azure Cosmos DB MongoDB database",
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/databaseNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "updateThroughputParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ThroughputSettingsUpdateParameters"
+            },
+            "description": "The RUs per second of the parameters to provide for the current MongoDB database."
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The RUs per second of the MongoDB database update operation will complete asynchronously."
+          },
+          "200": {
+            "description": "The RUs per second of the MongoDB database update operation was completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/ThroughputSettingsGetResults"
+            }
```

Comment:
no default response?

Reply:
note: just commenting on this, this is not a required change. There are several instances throughout this file. This is a correctness issue you guys should fix in a subsequent PR. You may have S360 items for this